src/index.js

```diff
@@ -88,7 +88,8 @@ export default function wdm(compiler, options = {}) {
       // eslint-disable-next-line no-param-reassign
       callback = callback || noop;
 
-      context.watching.close(callback);
+      callback();
+      // context.watching.close(callback);
```

Comment:
/cc @hiroppy The problem is here

Reply:
Good, so can we resolve this problem?

Reply:
I think bug in fsevents - https://github.com/fsevents/fsevents/issues/314, need to fix it, probably will have to figure it out yourself

Reply:
Maybe related - https://github.com/webpack/webpack/issues/10037

Reply:
Thank you for investigation!